editor/components/block-list/index.js

```diff
@@ -214,6 +252,35 @@ class BlockList extends Component {
 		}
 	}
 
+	onKeyDown( event ) {
+		const { onReplace, onChange } = this.props;
+
+		if ( ! this.shortcutTransforms ) {
+			return;
+		}
+
+		if ( ! isAccess( event ) ) {
```

Comment:
Any reason not to merge into previous condition?

Reply:
No :)